source/administration/object-management.rst

```diff
@@ -84,7 +84,7 @@ For a deeper discussion on the benefits of limiting prefix contents, see the art
 Object Versioning
 ----------------- 
 
-MinIO supports keeping multiple "versions" of an object in a single bucket. 
+MinIO supports keeping up to 10,000 "versions" of an object in a single bucket. 
```

Comment:
I suggest we preface this with a "Starting with", and perhaps add one sentence with something to the effect of "For workloads that require keeping more than 10K versions per object, please reach out to MinIO via hello@min.io"